lib/disqueue-client.js

```diff
@@ -237,8 +215,26 @@ DisqueClient.prototype.sendCommand = function ( commandCreated ) {
  * write
  *
  */
-DisqueClient.prototype.write = function ( collatedCommand ) {
-	this.netStream.write.apply( this.netStream, collatedCommand );
+var bufStar = Buffer.from('*', 'ascii');
```

Comment:
Can you move this on top. after the constants.

Reply:
Sure.  Will make the changes.

Reply:
Never mind.  Looks like you did it...